CriticalMass/ErrorStateModel.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ErrorStateModel.swift
+//  CriticalMaps
+//
+//  Created by Malte Bünz on 04.11.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+import Foundation
+
+struct ErrorStateModel {
+    let errorTitle: String
+    let errorMessage: String
+}
+
+extension ErrorStateModel {
+    static let `default` = ErrorStateModel(errorTitle: String.errorStateTitle,
```

Comment:
Super minor: Can we rename it to fallback instead of default. I would like to have specific error messages and embrace that int he API